components/browser/state/src/main/java/mozilla/components/browser/state/state/MediaState.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+/**
+ * Details about the last playing media in this tab.
+ *
+ * @property lastMediaUrl - [ContentState.url] when media started playing.
+ * This is not the URL of the media but of the page when media started.
+ * Defaults to "" (an empty String) if media hasn't started playing.
+ * @property lastMediaAccess The last time media started playing in the current web document.
+ * Defaults to [0] if media hasn't started playing.
+ */
+data class MediaState(
```

Comment:
Let's rename this one to `LastMediaAccessState` because we already have a `MediaSessionState` in `SessionState` and this could end up being where our future selves just add more and more media stuff and it doesn't belong here.